### PR TITLE
Add docs for command activity

### PR DESCRIPTION
**Pull request by dshokouhi**
Docs for: https://github.com/home-assistant/android/pull/1327

### CODE REVIEW

File: docs/notifications/commands.md [hunk 1]
```diff
@@ -116,6 +117,31 @@ automation:
...
+
+On Android you can send `message: command_navigate` to launch Google Maps to navigate to a location specified in the `title` of the notification. If the `title` is not set then the notification will post as normal. `title` will accept anything but to make the best use of it make sure to use either a real address, a place or `latitude,longitude` (in this order). More details can be found in [Google's documentation](https://developers.google.com/maps/documentation/urls/android-intents#launch-turn-by-turn-navigation). If you send invalid data then Google Maps may get stuck. Per the documentation Google Maps tries to make a best effort to select the first search result so if your `title` yields no results then Google Maps may appear stuck.
+
+User's can also optionally set `channel` to match the desired `mode` per the specification from Google, accepted values are `d, b, l, w` and will default to `d` for driving if not set or the value does not match any of the above.
+
+Example:
+
+```yaml
+automation:
+  - alias: Notify Mobile app
```
Comment (TomBrien) [inline docs/notifications/commands.md:132]:
```suggestion
  - alias: Mobile App Launch Navigate
```

**Review by TomBrien — CHANGES_REQUESTED** [review]
Just a super minor think, we should aim to keep the `alias`s unique I think

**Comment by dshokouhi** [comment]
> Just a super minor think, we should aim to keep the `alias`s unique I think

yea we got a few places to clean that up :)

**Review by TomBrien — APPROVED** [review]
All good by me

**Comment by dshokouhi** [comment]
Made some changes to reflect on the more generic approach here so had to shuffle it around.

File: docs/notifications/commands.md [hunk 2]
```diff
@@ -26,10 +27,53 @@ The Companion apps offer a lot of different notification options. In place of po
...
+
+```yaml
+automation:
+  - alias: Search google maps
+    trigger:
+      ...
+    action:
+      service: notify.mobile_app_<your_device_id_here>
+      data:
+        message: "command_navigate"
```
Comment (JBassett) [inline docs/notifications/commands.md:65]:
```suggestion
        message: "command_activity"
```

File: docs/notifications/commands.md [hunk 3]
```diff
@@ -26,10 +27,53 @@ The Companion apps offer a lot of different notification options. In place of po
...
+
+```yaml
+automation:
+  - alias: Navigate
+    trigger:
+      ...
+    action:
+      service: notify.mobile_app_<your_device_id_here>
+      data:
+        message: "command_navigate"
```
Comment (JBassett) [inline docs/notifications/commands.md:48]:
```suggestion
        message: "command_activity"
```

**Review by JBassett — CHANGES_REQUESTED** [review]

**Pull request merged**